lib/r10k/environment/with_modules.rb

```diff
@@ -126,13 +125,6 @@ def add_module(name, args)
 
   include R10K::Util::Purgeable
 
-  # Returns an array of the full paths that can be purged.
-  # @note This implements a required method for the Purgeable mixin
-  # @return [Array<String>]
-  def managed_directories
-    [@full_path]
-  end
-
```

Comment:
Note to self, this went away because the base environment now defines `managed_directories` as `[@full_path]` by default.